### ISSUE TITLE
Question: Start GUI to use apple menu with mouse

### ISSUE THREAD

**Issue by Arjay-El** [jkitchin/scimax#224]
OSX version:  Is it possible to start scimax in a gui mode with a menu having mouse support?

**Comment by jkitchin** [comment]
yes. I suppose it depends on how you start emacs, and where you got it from. The way I do that is described here: https://github.com/jkitchin/scimax#mac-users, and I use home brew for building emacs. It always opens in gui mode with mouse support for me. I start it from the command line with:

`emacs -q -l /path/to/scimax/init.el`

**Comment by Arjay-El** [comment]
Yes I use ./scimax inside terminal which contains that cmd sequence. However I can only use the scimax menus with key commands. Is that the expected behaviour?

**Comment by jkitchin** [comment]
No, it sounds like you are using an emacs that is compiled to run in the terminal. Where do you get your emacs from?

**Comment by Arjay-El** [comment]
I am using your distribution installed with the command you listed. I will uninstall and reinstall.

Sent from the Galaxy
-------- Original message --------From: John Kitchin <notifications@github.com> Date: 2018-08-13  8:01 PM  (GMT-05:00) To: jkitchin/scimax <scimax@noreply.github.com> Cc: Arjay-El <rjlilko@randing.ca>, Author <author@noreply.github.com> Subject: Re: [jkitchin/scimax] Question: Start GUI to use apple menu with mouse (#224) 
No, it sounds like you are using an emacs that is compiled to run in the terminal. Where do you get your emacs from?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/jkitchin/scimax","title":"jkitchin/scimax","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/jkitchin/scimax"}},"updates":{"snippets":[{"icon":"PERSON","message":"@jkitchin in #224: No, it sounds like you are using an emacs that is compiled to run in the terminal. Where do you get your emacs from?"}],"action":{"name":"View Issue","url":"https://github.com/jkitchin/scimax/issues/224#issuecomment-412706518"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/jkitchin/scimax/issues/224#issuecomment-412706518",
"url": "https://github.com/jkitchin/scimax/issues/224#issuecomment-412706518",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "Re: [jkitchin/scimax] Question: Start GUI to use apple menu with mouse (#224)",
"sections": [
{
"text": "",
"activityTitle": "**John Kitchin**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@jkitchin",
"facts": [

]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"jkitchin/scimax\",\n\"issueId\": 224,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"name": "Close issue",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueClose\",\n\"repositoryFullName\": \"jkitchin/scimax\",\n\"issueId\": 224\n}"
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/jkitchin/scimax/issues/224#issuecomment-412706518"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 367399820\n}"
}
],
"themeColor": "26292E"
}
]

**Comment by jkitchin** [comment]
That is not the problem for you. You are probably using the built in emacs. For mac OSX, scimax does not provide an emacs. Instead, you should follow these directions: https://github.com/jkitchin/scimax#mac-users

install your own emacs, and then use the scimax command.

**Comment by Arjay-El** [comment]
My initial install was with your first set of instructions. I am trying the second set using brew but emacs-plus is not recognized (see bottom of message). The installation recommends this, do I follow that instruction before cloning scimax?

Please try the Cask for a better-supported Cocoa version:
  brew cask install emacs

--- Message after using brew install method-------
Arjays-Air:~ arjay$ brew install emacs-plus
Error: No available formula with the name "emacs-plus" 
==> Searching for a previously deleted formula (in the last month)...
Warning: homebrew/core is shallow clone. To get complete history run:
  git -C "$(brew --repo homebrew/core)" fetch --unshallow

Error: No previously deleted formula found.
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching taps...
==> Searching taps on GitHub...
Error: No formulae found in taps.

**Comment by jkitchin** [comment]
Sorry, I left out a detail. See https://github.com/d12frosted/homebrew-emacs-plus

TL;DR: 
```
> brew tap d12frosted/emacs-plus
> brew install emacs-plus
```

**Comment by Arjay-El** [comment]
Using that extra command it loaded perfectly, complete with menus and mouse control. Thanks for your help and patience.

**Comment by jkitchin** [comment]
No worries, I hope you enjoy it more now!
